core/src/main/java/org/infinispan/CacheStream.java

```diff
@@ -171,6 +171,9 @@
     * {@link CacheStream#distributedBatchSize} value controls how many elements are processed per node at a time
     * when rehash is enabled. After those are complete the keys are sent to the originator to confirm that those were
     * processed.  If that node goes down during/before the response those keys will be processed a second time.</p>
+    * <p>It is possible to have the cache local to each node to be injected into this instance if the provided
+    * Consumer also implements the {@link org.infinispan.stream.CacheAware} interface.  This method will be invoked
+    * before the consumer apply method is invoked.</p>
```

Comment:
when referencing actual methods, it is better to follow them with the "()" and surround them with +. Also, the consumer method is +accept()+